### DISCUSSION TITLE
4gb Patch problem

### DISCUSSION THREAD

**Discussion by haloharry** [rebtd7/FF13Fix#71] [Q&A] [answered]
Hello, when i try to run the game after i apply the 4gb patch, I get this error (when i run from the EXE)
![image](https://user-images.githubusercontent.com/16374954/113195491-3e9b2900-925a-11eb-902e-d9881a96b2fa.png)
if i try to run it with steam i get this error 
![image](https://user-images.githubusercontent.com/16374954/113195547-51adf900-925a-11eb-81e3-7a817d38396a.png)
if i remove the 4gb patch, the game works.
Do you know anyway to fix this?

**Comment by Nucleoprotein** [comment, marked as answer]
Did you read the manual? https://github.com/rebtd7/FF13Fix#ff13
You clearly not have untouched.exe

**Reply by haloharry** [reply]
wow that you very much :3 I just thought he ment to rename it, so you have a back up.
I didn't think you Had to name it that.

**Comment by Reda-Ou** [comment]
I have this exact same issue even with untouched.exe
![image](https://github.com/user-attachments/assets/613f4f6e-1d1c-4f33-8805-e32d8425241a)

**Reply by Reda-Ou** [reply]
Using the Nova Crysalia mod launcher which also applies the 4gb works, but the proscribed 4gb patch method doesn't. /shrug
